include/mbedtls/config.h

```diff
@@ -2798,7 +2798,30 @@
 //#define MBEDTLS_SSL_CACHE_DEFAULT_MAX_ENTRIES      50 /**< Maximum entries in cache */
 
 /* SSL options */
-//#define MBEDTLS_SSL_MAX_CONTENT_LEN             16384 /**< Maxium fragment length in bytes, determines the size of each of the two internal I/O buffers */
+//#define MBEDTLS_SSL_MAX_CONTENT_LEN             16384 /**< Maximum fragment length in bytes, determines the size of each of the two internal I/O buffers */
+
+//#define MBEDTLS_SSL_IN_CONTENT_LEN              16384 /**< Optionally override MBEDTLS_SSL_MAX_CONTENT_LEN for the inward TLS buffer only. */
+
+/** \def MBEDTLS_SSL_OUT_CONTENT_LEN
+ *
+ * Optionally override MBEDTLS_SSL_MAX_CONTENT_LEN for the outward TLS buffer only.
```

Comment:
Documentation: This is a fragment, and it needs a verb! Suggestion:
`This option can override MBEDTLS_SSL_MAX_CONTENT_LEN for the outward TLS buffer only.`

Reply:
“override” is a verb. If you change to add the subject “this option” then don't use “can” since this option _does_ override `MBEDTLS_SSL_MAX_CONTENT_LEN`.

Reply:
But it isn't clear what “optionally” means here. My guess is that this means “if the line containing the `#define` directive is uncommented”, but I shouldn't have to guess. Elsewhere in the file we write “Uncomment to …”, please follow this style if you need to express that something isn't active by default. For the header of the description, though, just say what the feature does, and don't require the reader to know about another feature: “The size of the outward TLS buffer”.